src/api/app/controllers/webui/code_of_conduct_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Webui::CodeOfConductController < Webui::WebuiController
+  def index
+    @code_of_conduct = ::Configuration.first.code_of_conduct
+
+    head :not_found if @code_of_conduct.blank?
```

Comment:
If you just throw an `ActiveRecord::RecordNotFound` it'll do the right thing...

Reply:
Done. :+1: